### PR TITLE
Add Celery health check endpoint 

### PR DESCRIPTION
**Pull request by sudan45**
Addresses 
- https://github.com/the-deep/server/issues/1512

## Feature:
- Introduced a health check endpoint (`HealthCheckView`) that allows clients to verify the status of the Celery worker.
- The endpoint responds with a "healthy" status when the worker is active and responsive, and "unhealthy" if not.

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [x] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

**Comment by thenav56** [comment]
Let's explore this https://github.com/revsys/django-health-check
It already has ping function https://github.com/revsys/django-health-check/blob/master/health_check/contrib/celery_ping/backends.py

File: deep/urls.py [hunk 1]
```diff
@@ -619,6 +619,8 @@ def get_api_path(path):
             name="favicon"),
 
     re_path('ses-bounce/?$', ses_bounce_handler_view, name='ses_bounce'),
+    re_path(r'ht/', include('health_check.urls')),
```
Comment (susilnem) [inline deep/urls.py]:
```suggestion
    re_path(r'health-check/', include('health_check.urls')),
```

**Review by susilnem — CHANGES_REQUESTED** [review]
Minor change

**Review by susilnem — APPROVED** [review]
LGTM!

**Pull request merged**